cdk/dcm_floodlight/transform.js

```diff
@@ -57,7 +60,34 @@ function trackPostMapper(input, mappedPayload, rudderContext) {
 
   const baseEndpoint = "https://ad.doubleclick.net/ddm/activity/";
 
-  let event = getValueFromMessage(message, "event");
+  let event;
+  if (message.type === "page") {
+    const { category } = message.properties;
+    const { name } = message || message.properties;
+
+    if (!category && !name) {
```

Comment:
Why cant we support unnaamed and non categorised page view events? Something like `Viewed a Page`?

Reply:
Viewed a page won't be mapped to any event in the config. Generally, a user provides the event/action name to a floodlight activity.

Reply:
But the user can map `Viewed Page` as a conversion event right?

Reply:
Makes sense. Added this enhancement